### PR TITLE
Fixed job name and event name dropdown

### LINKED ISSUE #382: Show event suggestion as dropdown
volunteer/report/

### PR DESCRIPTION
**Pull request by anweshknayak**
closes #382, #383.

screenshot from 2016-11-27 02 52 36

### CODE REVIEW

File: vms/volunteer/views.py [hunk 1]
```diff
@@ -149,8 +149,8 @@ class ShowReportListView(LoginRequiredMixin, ListView):
 
     def post(self, request, *args, **kwargs):
         volunteer_id = self.kwargs['volunteer_id']
-        event_list = get_signed_up_events_for_volunteer(volunteer_id)
-        job_list = get_signed_up_jobs_for_volunteer(volunteer_id)
+        event_list = get_events_ordered_by_name()
+        job_list = get_jobs_ordered_by_title()
```
Comment (tapasweni-pathak) [inline vms/volunteer/views.py]: Previously event_list was having the events for which the volunteer has signed up for, now after your change it will have all the events ordered by name. Does doing this changed helped you in something else? Why this change?

Comment (anweshknayak) [inline vms/volunteer/views.py]: I will take a look again. I think I had some misunderstanding with the code base. I think I should close this pull request.

Comment (tapasweni-pathak) [inline vms/volunteer/views.py]: Sure. Let us know if you get stuck and need help.
